packages/js/product-editor/src/components/variations-table/variation-actions-menus/variation-actions.tsx

```diff
@@ -95,8 +93,9 @@ export function VariationActions( {
 				/>
 			</MenuGroup>
 			<VariationQuickUpdateMenuItem.Slot
+				name=""
 				group={ 'top-level' }
-				onChange={ onChange }
+				onChange={ ( value ) => onChange( value ) }
```

Comment:
This change can also be reverted I believe.

Reply:
Fixed with 6bcf79b8cf3d057364541118c6c4fe8f3b03dc8d